web_action_conditionable/README.rst

```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```

Comment:
Hi @kossovo thanks for the contribution
Please follow the newest OCA README template:
https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
You should have a document title (add ======================== before)

"This module extends the functionality of actions in tree views."
s/supports/supports